src/main/steps/index.ts

```diff
@@ -47,10 +47,10 @@ export const getNextIncompleteStepUrl = (req: AppRequest): string => {
   return `${url}${queryString}`;
 };
 
-export const getNextStepUrl = (req: AppRequest): string => {
+export const getNextStepUrl = (req: AppRequest, data: Partial<Case>): string => {
   const { path, queryString } = getPathAndQueryString(req);
   const nextStep = sequence.find(s => s.url === path);
-  const url = nextStep ? nextStep.getNextStep(req.body) : CHECK_ANSWERS_URL;
+  const url = nextStep ? nextStep.getNextStep(data) : CHECK_ANSWERS_URL;
```

Comment:
Looking at the tests, I don't think there's a need to change from `req.body` and add this unnecessary `data` param as it should always exist on `req` right?

Reply:
i guess but it depends on the parsedbody return